source/images/buyers-guide/icon-face-neutral.svg

```diff
@@ -1 +1,9 @@
-<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 24 24" xml:space="preserve"><style>.st25{fill:#4cbfa9}</style><path class="st25" d="M12 0C5.4 0 0 5.4 0 12s5.4 12 12 12 12-5.4 12-12S18.6 0 12 0zm0 22.5C6.2 22.5 1.5 17.8 1.5 12S6.2 1.5 12 1.5 22.5 6.2 22.5 12 17.8 22.5 12 22.5z"/><path class="st25" d="M7.9 11.1c.7 0 1.2-.5 1.2-1.2s-.6-1.1-1.2-1.1-1.2.5-1.2 1.1.5 1.2 1.2 1.2zm0-1.2v.1s-.1 0 0-.1zM16.1 11.1c.7 0 1.2-.5 1.2-1.2s-.5-1.2-1.2-1.2-1.2.5-1.2 1.2.6 1.2 1.2 1.2zm0-1.2v.1s-.1 0 0-.1zM17.2 15.5H6.8c-.3 0-.6.3-.6.6s.3.6.6.6h10.3c.3 0 .6-.3.6-.6s-.2-.6-.5-.6z"/></svg>
\ No newline at end of file
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```

Comment:
this file looks like it needs an `svgo` treatment.

Reply:
Ummm... I did... using the same configs as what we have in `package.json` 

> `svgo --multipass --pretty --enable=removeTitle`

Reply:
ah, I've been running it without arguments, didn't really we have --pretty turned on in the package.json